application/controllers/UserManagementController.php

```diff
@@ -772,7 +776,8 @@ public function actionImportUsers(string $importFormat = 'csv'): string
                 break;
             case "csv":
             default:
-                $aNewUsers = UserParser::getDataFromCSV($_FILES); //importFormat default is csv ...
+                $aNewUsers = UserParser::getDataFromCSV($_FILES);
+                $importFormat = 'csv'; /* Fix usage of importFormat in view */
```

Comment:
Is this needed now?

Reply:
It's in https://github.com/LimeSurvey/LimeSurvey/blob/837eae16db807db3526a2439420a1ceecd492ec6/application/controllers/UserManagementController.php#L762 import function

Not needed

Reply:
But, yes. It is needed. In case theyinput something not empty and not CSV neither json.

Reply:
> But, yes. It is needed. In case theyinput something not empty and not CSV neither json.

No !

1. It's in imported file function : https://github.com/LimeSurvey/LimeSurvey/blob/e8e46e0007f7cf26aecb2ddc70de2eb3d35390ce/application/controllers/UserManagementController.php#L762 
2. It's not related to this issue
3. It break at https://github.com/LimeSurvey/LimeSurvey/blob/e8e46e0007f7cf26aecb2ddc70de2eb3d35390ce/application/controllers/UserManagementController.php#L782 or https://github.com/LimeSurvey/LimeSurvey/blob/d23d48c2592ee69792ba1eceb89016d1d7667f0d/application/models/UserParser.php#L17

Reply:
You ask if it's needed, i look : no seems not needed, i remove … you ask to add it again …

Reply:
>  you ask to add it again …

No, After I said it was not needed, I thought I saw something that make it look like needed.
It is not that I asked explicitely to add it again :)

It is already approved